Openstack log support (#9116)
DataDog/integrations-core: openstack/README.md

2. [Restart the Agent][5].

##### Log collection

> 1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent, you can enable it in `datadog.yaml`:
```

   ```yaml
   logs_enabled: true
   ```

> 2. Add this configuration block to your `openstack.d/conf.yaml` file to start collecting your Openstack Logs:

Reviewer: ```suggestion
2. Add this configuration block to your `openstack.d/conf.yaml` file to start collecting your Openstack logs:
```

   ```yaml
   logs:
     - type: file
       path: "<LOG_FILE_PATH>"
       source: openstack

>        service: "<SERVICE_NAME>"

Reviewer 2: ```suggestion
```

service is optional

   ```

>     Change the `path` and `service` parameter values and configure them for your environment. See the [sample openstack.d/conf.yaml][4] for all available configuration options.

Reviewer 2: ```suggestion
    Change the `path` parameter value and configure them for your environment. See the [sample openstack.d/conf.yaml][4] for all available configuration options.
```
